src/Maker/MakeUser.php

```diff
@@ -114,6 +115,12 @@ class_exists(DoctrineBundle::class)
 
     public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator): void
     {
+        if ($input->getOption('uuid_id')) {
```

Comment:
Let's move this conditional to the `interact` method - it's a better dx when we know right away if we're missing a key package.